src/sfctl/apiclient.py

```diff
@@ -20,9 +20,11 @@ def create(_):
     endpoint = client_endpoint()
 
     if not endpoint:
-        raise CLIError("Connection endpoint not found. "
-                       "Before running sfctl commands, connect to a cluster using "
-                       "the 'sfctl cluster select' command.")
+        raise CLIError('Connection endpoint not found. '
```

Comment:
Could this be confusing at all if we make the default endpoint localhost:19080 when you dont provide a endpoint?

Reply:
Users will still need to select a cluster first, the default is only for not explicitly providing the endpoint on the select command. Will make help text more clear.